myocamlbuild.ml

```diff
@@ -941,7 +941,7 @@ dispatch
                         "gettext" 
                         (BaseEnvLight.load
                            ~allow_empty:true
-                           ~filename:(Lazy.force MyOCamlbuildBase.env_filename)
+                           ~filename:(MyOCamlbuildBase.env_filename)
```

Comment:
Yes, and I guess this explains it. I was using oasis-0.4.5 as released.

So, what's the right bootstrapping procedure then?